website/docs/cookbooks/scala-docker.md

```diff
@@ -18,7 +18,7 @@ Passing `--docker` to the `package` sub-command generates a Docker image. When c
 The following command generates a `hello-docker` image with the `latest` tag:
 
 ```bash
-scala-cli package --docker HelloDocker.scala --docker-image-repository hello-docker
+scala-cli package --jvm 8 --docker HelloDocker.scala --docker-image-repository hello-docker
```

Comment:
Do we really need to pass that option here? It should work fine without it (doesn't it?)